src/app/portlets/dot-edit-page/layout/dot-edit-layout-designer/components/dot-layout-designer/dot-layout-designer.component.html

```diff
@@ -6,7 +6,7 @@
         <div
             class="dot-layout-designer__sidebar--left"
             [ngClass]="'dot-layout-designer__sidebar--' + group.get('sidebar').value['width']"
-            *ngIf="group.get('sidebar').value['location'] === 'left'"
+            *ngIf="group.get('sidebar')?.value?.location === 'left'"
```

Comment:
Is there any chance that `group.get('sidebar')` might be an empty object or undefined? If empty object then optional chaining must be only in `value` e.g. `group.get('sidebar').value?.location`, if undefined then leave as is.